tools/create-app.bash

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
 
-# Create a script that given an output directory builds the current application
-# and generates an .aab file in the output dir.
-
 ### Parameters ###
 
 OUTPUT_DIRECTORY=$1
 ASSETS_DIRECTORY=$2
 SPLASH_BACKGROUND_COLOR=$3
+KEYSTORE_DIRECTORY=$4
```

Comment:
One last little thing. I've tried the script and thought that this corresponds to the keystore directory, but in fact it needs the keystore file to be assigned. I'd update the variable name to `KEYSTORE_PATH` instead, and we're done.